client/src/components/ForceGraph.vue

```diff
@@ -468,16 +468,17 @@ export default {
       );
       this.selections.graph.attr("transform", transform);
 
+      // Update- Sometimes it somehow hides part of the graph
       // Define some world boundaries based on the graph total size
       // so we don't scroll indefinitely
-      const graphBox = this.selections.graph.node().getBBox();
-      const margin = 200;
-      const worldTopLeft = [graphBox.x - margin, graphBox.y - margin];
-      const worldBottomRight = [
-        graphBox.x + graphBox.width + margin,
-        graphBox.y + graphBox.height + margin
-      ];
-      this.zoom.translateExtent([worldTopLeft, worldBottomRight]);
+      // const graphBox = this.selections.graph.node().getBBox();
```

Comment:
Why are you commenting it out? If that doesnt work consider removing that completly from master

Reply:
I was going to try fixing it, but removing it for good seems like a better option